tests/net/all/prj.conf

```diff
@@ -394,6 +394,16 @@ CONFIG_NET_DEBUG_L2_IEEE802154_DISPLAY_PACKET=y
 CONFIG_NET_DEBUG_L2_IEEE802154_DISPLAY_PACKET_RX=y
 CONFIG_NET_DEBUG_L2_IEEE802154_DISPLAY_PACKET_TX=y
 
+# Shell MQTT backend
+CONFIG_LOG=y
+CONFIG_SHELL=y
+CONFIG_SHELL_BACKEND_MQTT=y
+CONFIG_SHELL_MQTT_SERVER_ADDR="www.zephyrproject.org"
```

Comment:
It should probably default to an actual MQTT server, something like `test.mosquitto.org` maybe